CMakeLists.txt

```diff
@@ -100,6 +100,8 @@ if (MSVC)
   add_definitions(-D_WIN32_WINNT=0x600 -D_SCL_SECURE_NO_WARNINGS -D_CRT_SECURE_NO_WARNINGS -D_WINSOCK_DEPRECATED_NO_WARNINGS)
   # needed to compile protobuf
   add_definitions(/wd4065 /wd4506)
+  # TODO(jtattermusch): revisit warnings disabled when upgrading to protobuf3.6.0
```

Comment:
When upgrading to 3.6? Isn't it what this PR is about?

Reply:
Rephrased.